README.md

```diff
@@ -15,11 +15,26 @@ For detailed requirements and install instructions see [irkernel.github.io](http
 We provide Windows and Mac OS X binary packages of all the needed packages:
 
 ```r
-install.packages(c('rzmq','repr','IRkernel','IRdisplay'),
+install.packages(c(repr','IRkernel','IRdisplay'),
                  repos = c('http://irkernel.github.io/', getOption('repos')))
 IRkernel::installspec()
 ```
 
+Per default `IRkernel::installspec()` will install a kernel with the name `"ir"` and a
+display name of `"R"`. Multiple calls will overwrite the kernel with a kernel spec pointing to the last
+R interpreter you called that commands from. You can install kernels for multiple versions of R
+by supplying a `name` and `displayname` argument to the `installspec()` call (You still need to
+install these packages in all interpreter you want to run as a jupyter kernel!):
+
+```r
+# in R 3.2.2
+IRkernel::installspec(name="ir322", displayname="R 3.2.2")
+# in R 3.1
+IRkernel::installspec(name="ir31", displayname="R 3.1")
```

Comment:
guess I'm also to python centric... No spaces around "="  in arguments to a function call...